geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/conf/ConfigLoader.scala

```diff
@@ -0,0 +1,69 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.conf
+
+import scala.xml.XML
+import java.io.FileNotFoundException
+
+import com.typesafe.scalalogging.LazyLogging
+
+object ConfigLoader extends LazyLogging {
+  val GEOMESA_CONFIG_FILE = "geomesa.config.file"
+  val GEOMESA_CONFIG_FILE_NAME = "geomesa-site.xml"
+  val GEOMESA_EMBEDDED_CONFIG_FILE_PATH = "/org/locationtech/geomesa/geomesa-site.xml.template"
+  var isLoaded: Boolean = false
```

Comment:
yeah, i was thinking use a singleton object that loads the xml in it's constructor and skip the isLoaded check.